### ISSUE TITLE
Fix 'test_providers/test_providers_no_os' tests in 4.2

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh-qa#1561]
### Issue information

| Related issue |
|--|
| Closes https://github.com/wazuh/wazuh-qa/issues/1531|

When it comes to starting with the test fix, there is some information that can help you:
```
Module: Vulnerability Detector
Support Team:  "binary beasts"
Target: Manager   
OS: Linux
```
I add some information obtained while researched the logs that failed.

## Case:

|  Type | Description  |
|--|--|
|Test Path | test_providers/test_providers_no_os
| Consistent | no
| Test Execution | 1/3 executions failed
| Cases Fails | 5
| Summary | On `pytest_runtest_setup` when check `host_type`.
| Error message | "ERROR at setup of test_providers_no_os"

In order to finish this issue the following tasks should be fulfilled:
- [ ] Research of fails.
- [ ] Apply Fix 
- [ ] Full Green/ Full Yellow  in `test_providers/test_providers_no_os` for 3 times. 
- [ ] Documentation of any important change done for these tests or the used tools.

**Comment by pereyra-m** [comment]
Results for `test_providers/test_providers_no_os`

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[centos_providers_no_os_1.log](https://github.com/wazuh/wazuh-qa/files/6812196/centos_providers_no_os_1.log)| 2021-07-13| Matias | :green_circle:
| [centos_providers_no_os_2.log](https://github.com/wazuh/wazuh-qa/files/6812195/centos_providers_no_os_2.log)| 2021-07-13| Matias | :green_circle:
| [centos_providers_no_os_3.log](https://github.com/wazuh/wazuh-qa/files/6812194/centos_providers_no_os_3.log)| 2021-07-13| Matias | :green_circle:

**Comment by mdengra** [comment]
### 2021-07-14

Test results:

| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[test_providers_no_os_local_r1.log](https://github.com/wazuh/wazuh-qa/files/6816835/test_providers_no_os_local_r1.log)| 2021-07-14 | Miguel | :yellow_circle: 
|[test_providers_no_os_local_r2.log](https://github.com/wazuh/wazuh-qa/files/6816838/test_providers_no_os_local_r2.log)| 2021-07-14 | Miguel | :yellow_circle: 
|[test_providers_no_os_local_r3.log](https://github.com/wazuh/wazuh-qa/files/6816839/test_providers_no_os_local_r3.log)| 2021-07-14 | Miguel | :yellow_circle:

**Comment by damarisg** [comment]
Results when disabling modules: `sca`, `syscollector` and `rootcheck`.


| Test Executions | Date  | By  | Status | 
|--|--|--|--|
|[ProviderNoOs.log](https://github.com/wazuh/wazuh-qa/files/6819893/ProviderNoOs.log)| 2021-07-15| Seyla| 🟢 
|[ProviderNoOs1.log](https://github.com/wazuh/wazuh-qa/files/6819894/ProviderNoOs1.log)| 2021-07-15| Seyla| 🟡 
| [ProviderNoOs2.log](https://github.com/wazuh/wazuh-qa/files/6819895/ProviderNoOs2.log)| 2021-07-15| Seyla| 🟢 


| Reference | Status | 
|--|--|
| 🟢 | Pass without warnings|
| 🟡 | Pass with Warnings |

**Comment by damarisg** [comment]
This issue is closed, after executes for a lot of times and see that works. The solved applied was disabling `sca`, `syscollector` and `rootcheck` modules.

**Comment by damarisg** [comment]
This issue is closed by #1605 

## Test Results from `1531-full-yellow-vuln-det`

| Test Executions | Tests | Date  | By  | Status | 
|--|--|--|--|--|
|[YellowResultsProvidersNoOs.log](https://github.com/wazuh/wazuh-qa/files/6857565/YellowResultsProvidersNoOs.log)
| test_providers/test_providers_no_os |2021-07-21| Seyla| 🟡 
| [YellowResultsProvidersNoOs2.log](https://github.com/wazuh/wazuh-qa/files/6857574/YellowResultsProvidersNoOs2.log)
|test_providers/test_providers_no_os |2021-07-21| Seyla| 🟡 
|[YellowResultsProvidersNoOs3.log](https://github.com/wazuh/wazuh-qa/files/6857579/YellowResultsProvidersNoOs3.log)
 |test_providers/test_providers_no_os| 2021-07-21| Seyla|  🟡 

| Reference | Status | 
|--|--|
| 🟡 | Pass with Warnings |

**Comment by damarisg** [comment]
This issue is closed because the test was disabled by #1602.
